docs/UserGuide.md

```diff
@@ -278,7 +278,8 @@ Format: `prioritize INDEX p/PRIORITY`
 You can set priority levels for assignments based on their urgency. Assignments tagged with a priority level will be
 shown with a coloured priority tag (As shown highlighted in red in the figure below).
 
-DIAGRAM
+   ![UserGuidePriorityTags](images/UserGuidePriorityTags.PNG)
+   *Figure X: The priority tags of assignments highlighted in red*
```

Comment:
The diagram looks good! There seems to be a problem with the formatting of the Figure caption though (Refer to screenshot below)

![image](https://user-images.githubusercontent.com/58943071/97591539-aeeb1780-1a3a-11eb-8139-fd66fbb7efd9.png)